### ISSUE TITLE
[BUG] - Transcoding broken as of v3.5.7

### ISSUE THREAD

**Issue by picklepopper1** [CappielloAntonio/tempo#95]
## Describe the bug
Transcoding fails to happen no matter what the settings. I prefer that the server decides transcoding rules, but even disabling that setting does not lead to the app transcoding at the desired bitrate/format. This has superseded uninstall/reinstall

## Expected behavior
When playing back audio, I expect the app to request transcoding from the server at ~256kb/s in the opus format.

## To Reproduce
1. Go to any page
2. Click on any song
3. Pull up now playing screen
4. See app streaming at original quality in FLAC encoding on the top of now playing screen

## Environment
 - Android device: Pixel 7 Pro
 - Android OS version: 14 QPR1
 - App version: v3.5.7
 - Other relevant details: Any kind of transcoding settings turned on in the app. Doesn't matter connection type.

## Logs or Screenshots
I apologize for not knowing how to get logs from the app. Screenshots would not be relevant here. 

## Additional context
I don't recall having this issue on the version prior. There have been no server updates (Navidrome) that could have facilitated a change.

**Comment by picklepopper1** [comment]
![I did find a relevant screenshot I could share.](https://github.com/CappielloAntonio/tempo/assets/79949408/6b4b3210-06af-4a3f-95cb-75c109d59505)

**Comment by CappielloAntonio** [comment]
Hi @picklepopper1, as an app all I do is show what the server is sending me. I would rather make sure that the server is sending me the correct format (and from the image it appears so) with the correct bitrate. 

I don't rule out that this is what is really happening, as the value of transcodedSuffix appears to be correct (opus extension). It just seems like the information shown is inconsistent with what you're actually hearing, but there's little I can do about it as as a client I only read the information that the server sends me.

**Comment by adnigcx** [comment]
Hello!

First of all I really enjoy the app and I use it primarily with my dockerized Navidrome!

The issue persists if on WiFi oder 4G.

Thank you for your hard work!

However, since the newest version, the same as [picklepopper1](https://github.com/picklepopper1) issued above, the transcoding is not working anymore.
![Tempo_1](https://github.com/CappielloAntonio/tempo/assets/12124002/765083c6-2bb0-4e04-b06a-82c09b94f871)

![Tempo_2](https://github.com/CappielloAntonio/tempo/assets/12124002/c1fcf18f-8103-465a-bfa5-16d7a52dd301)

![Temp_3](https://github.com/CappielloAntonio/tempo/assets/12124002/6627a79c-ef3a-4515-af90-20b7a04a4003)

**Comment by CappielloAntonio** [comment]
Hi @adnigcx, as I was saying Navidrome doesn't send information about the transcoded file, but that doesn't mean the file isn't transcoded. You can check the docker logs and check that the request coming to the server is correct and you can download the file obtained from the previous request to make sure the file is transcoded.

In this new iteration nothing has changed other than the values shown in the interface which I had to summarize as what was shown was not always truthful (see when a transcoding that was not supported by the server was requested, or the decision-making power over transcoding was left to the server).

**Comment by GallowsDove** [comment]
I think the main issue is that the disclaimer on top is too wordy and not clear enough for a lot of users, it could be somehow reworded to say basically "This is information about the file on the server, the sound you're hearing is probably transcoded to (transcode settings)

**Comment by CappielloAntonio** [comment]
I think a possible solution could be to warn that a possible transcoding is in progress and let the info dialog explain what's happening. Unfortunately you cannot rely on the transcoding options you choose, because everything depends on the server.

![Screenshot_20231105-174635_Tempo](https://github.com/CappielloAntonio/tempo/assets/14802539/689ab5f7-a9bd-42e3-8794-d32431d9b6da)

**Comment by CappielloAntonio** [comment]
A new version is being released with the aforementioned.
